client/src/api/auth.ts

```diff
@@ -0,0 +1,53 @@
+import { fetcher } from './fetcher';
+
+export type accessToken = {
+  accessToken: string;
+};
+
+const GOOGLE_CLIENT_ID = process.env.REACT_APP_GOOGLE_CLIENT_ID || '';
```

Comment:
p3
개인적으로 너무 길면 읽기 귀찮아서 간소화하는거 어떤지 제안드립니다~
```suggestion
const GOOGLE_CLIENT_ID = process.env.GOOGLE_CLIENT_ID || '';
```